beast/plotting/plot_ast_histogram.py

```diff
@@ -30,83 +30,85 @@ def plot_ast(ast_file, sed_grid_file=None):
     """
 
     # read in AST info
-    ast_table = Table.read(ast_file, format='ascii')
-    ast_fluxbins = Table.read(ast_file.replace('inputAST','ASTfluxbins'), format='ascii')
+    ast_table = Table.read(ast_file, format="ascii")
+    ast_fluxbins = Table.read(
+        ast_file.replace("inputAST", "ASTfluxbins"), format="ascii"
+    )
 
     # get filter names (and it's even in order!)
-    filter_cols = [col for col in ast_table.colnames if '_' in col]
+    filter_cols = [col for col in ast_table.colnames if "_" in col]
     filter_list = [col[-5:] for col in filter_cols]
     n_filter = len(filter_list)
 
     # if chosen, read in model grid
     if sed_grid_file is not None:
-        modelsedgrid = FileSEDGrid(sed_grid_file)
+        modelsedgrid = SEDGrid(sed_grid_file)
         with Vega() as v:
             _, vega_flux, _ = v.getFlux(filter_cols)
         sedsMags = -2.5 * np.log10(modelsedgrid.seds[:] / vega_flux)
-        good_seds = np.where(modelsedgrid.grid['logL'] > -9)[0]
-
+        good_seds = np.where(modelsedgrid.grid["logL"] > -9)[0]
```

Comment:
We don't need this when we apply this cut while generating a SpectralGrid.

Reply:
Good point.

Reply:
Removed this line and the use of `good_seds` in one place.